mizar/common/rpc.py

```diff
@@ -255,6 +278,19 @@ def delete_ep(self, ep):
         logger.info(log_string)
         returncode, text = run_cmd(cmd)
         logger.info("returns {} {}".format(returncode, text))
+        self.delete_packet_metadata(ep)
+
+    def delete_packet_metadata(self, ep):
+        jsonconf = {
+            "tunnel_id": ep.get_tunnel_id(),
```

Comment:
Does referencing ep properties after it has been deleted not pose a problem?

Reply:
delete_ep is to delete ep data in xdp table. Here ep object is still there to provide properties.